internal/kuberesource/parts.go

```diff
@@ -47,7 +47,7 @@ func NodeInstaller(namespace string) *NodeInstallerConfig {
 					WithHostPID(true).
 					WithInitContainers(Container().
 						WithName("installer").
-						WithImage("ghcr.io/edgelesssys/contrast/node-installer:latest").
+						WithImage("ghcr.io/edgelesssys/contrast/node-installer-microsoft:latest").
```

Comment:
One thing that is still unclear to me is how we gonna know which node-installer-image supports which platform variant.

Reply:
Microsoft -> AKS
Kata -> Bare-Metal

Or what's the question?

Reply:
Kata -> K3sQEMUTDX + RKE2QEMUTDX, as there is no "bare-metal" platform.

Reply:
There will only be a Single bare-Metal Node-installer though